source/screens/caseScreens/CaseSummary.tsx

```diff
@@ -178,10 +178,16 @@ const computeCaseCardComponent = (
       } ${getSwedishMonthNameByTimeStamp(payments.payment.givedate, true)}`
     : undefined;
 
-  const unApprovedCompletionDescriptions: string[] =
-    statusType.includes("completion") || statusType.includes("randomCheck")
-      ? getUnapprovedCompletionDescriptions(completions)
-      : [];
+  const isStatusCompletion = [
+    "completion",
```

Comment:
I suggest that you put these in constants

Reply:
Will be fixed in a later PR :)